deploy/vagrant/create

```diff
@@ -87,11 +87,10 @@ done
 echo "TachyonMaster" >> files/workers
 
 # create virtual machine without provisioning
-[[ "$PROVIDER" == "aws" ]] && sgid=`python bin/init_aws.py | tail -n 1`
+[[ "$PROVIDER" == "aws" ]] && sgid=$(python bin/init_aws.py | tail -n 1)
 # pass security group id to vagrant up
 export AWS_SECURITY_GROUP_ID_TACH=$sgid
-IS_SPOT=`python bin/is_aws_spot.py`
-if [[ "$PROVIDER" == "aws" && "$IS_SPOT" == 0 ]]; then
+if [[ "$PROVIDER" == "aws" && "$IS_SPOT" == "$(python bin/is_aws_spot.py)" ]]; then
```

Comment:
where did this $IS_SPOT come from?
I assume you meant `&& "0" == "$(python bin/is_aws_spot.py)"`?